README.md

```diff
@@ -7,40 +7,51 @@ Ferroamp MQTT support sends updates to these topics:
  * extapi/data/eso (interval 5s)
  * extapi/data/sso (interval 5s)
  * extapi/data/esm (interval 60s)
- 
+
 ## Prerequisites
 - Home assistant `0.115.0`
-- Enable Ferroamp MQTT by contacting Ferroamp Support and to get the username and password for your Energy MQTT broker.
-- Enable MQTT in Home assistant and set the broker to your Ferroamp Energy IP and configure it with your username and password received from Ferroamp.
+- Enable Ferroamp MQTT by contacting [Ferroamp Support](https://ferroamp.com/sv/kontakt/) to get the username and password for your Energy MQTT broker.
+- Enable MQTT in Home assistant and set the broker to your Ferroamp Energy IP and configure it with your username and password received from Ferroamp (or setup a bridge-connection if you already have an MQTT-server, example for [Mosquitto here](https://selfhostedhome.com/using-two-mqtt-brokers-with-mqtt-broker-bridging/)).
 
 ### Git installation
 
 1. Make sure you have git installed on your machine.
 2. Navigate to you home assistant configuration folder.
-3. Create a `custom_components` folder of it does not exist, and cd into it.
-4. Execute the following command: `git clone https://github.com/henricm/ha-ferroamp.git ferroamp`
+3. Execute the following command: `git clone https://github.com/henricm/ha-ferroamp.git`
+4. Create a `custom_components` folder if it does not exist, and cd into it.
+5. Create a symbolic link by executing `ln -s ../ha-ferroamp/custom_components/ferroamp`
 
 ## Setup
 
-1. Add a sensor for the ferroamp platform to your `<config dir>/configuration.yaml`
-
-```yaml
-sensor:
-  - platform: ferroamp
-```
+1. Add the `Ferroamp MQTT Sensors`-integration
+2. Set a name for the integration as well as the MQTT-prefix where updates are sent (default values are probably fine for a standard-setup but if a bridge-connection is used the MQTT-topics can be re-mapped)
+3. Wait for all the devices to become present (EnergyHub, SSO's, ESOs and ESMs depending on your setup. Be patient since ESMs are only updated every 60 seconds.)
 
-There are currently no configuration options. This integration will add some sensors with the prefix `ferroamp_`. For the SSO and ESO sensors, they will include the id of the SSO or ESO unit, eg `sensor.ferroamp_sso_123456_pv_string_power`. 
+This integration will add some sensors with the prefix `<integration name>_`. For the SSO and ESO sensors, they will include the id of the SSO or ESO unit, eg `sensor.ferroamp_sso_123456_pv_string_power`.
 
 I'm also still figuring out what some of the sensors actually are, since the Ferroamp API documentaiton still seems to be incomplete in some areas.
 
+## Upgrading from a version before config flow was implemented
+
+1. Remove the integration configuration. i.e this
+   ```yaml
+   sensor:
+     - platform: ferroamp
+   ```
+2. Restart Home Assistant
+3. Remove all Ferroamp entities (they will be re-created once the new setup is completed)
```

Comment:
Will this make all history for those entities go away?

Reply:
I think that as long as the entity IDs remain the same, the history will stay. At least that's what I have seen in my test environment. And I think that most of the sensors have retained their IDs.